spec.bs

```diff
@@ -116,6 +132,49 @@ if (document.wasDiscarded) {
 }
 </pre>
 
+Feature Policies {#feature-policies}
+=====================
+To improve CPU utilization of [=nested browsing contexts=] that are not critical to a page's usability two feature policies are defined:
+ * execution-while-not-rendered, which has a [=default allowlist=] of '*'
+ * execution-while-out-of-viewport, which has a [=default allowlist=] of '*'
+
+The <dfn export>execution-while-not-rendered</dfn> policy controls whether [=task source/freezable=]
+tasks should execute for [=nested browsing contexts=] whose [=browsing context container=] is not [=being rendered=]. ie. `display: none`.
+
+The <dfn export>execution-while-out-of-viewport</dfn> policy controls whether [=task source/freezable=]
+tasks should execute for [=nested browsing contexts=] whose [=browsing context container=] does not intersect the [=viewport=] according
+to [=compute the intersection of a target element and the root=].
+
+Application of these policies is accomplished by placing the document (and its decendants) in a [=/frozen=] state when the following
+conditions have been met:
+ * [=iframe load event steps=] have been run
+ * the policy is disabled for the document
+ * relevant policy condition applies (not rendered or scrolled out of view)
```

Comment:
Elsewhere in this spec, I'm seeing "NOTE: background tabs that are actively doing work on behalf of the user (eg. playing audio) are generally not frozen or discarded.". Are these features explicitly override this statement? Maybe in the context of an iframe?

Reply:
Yes the do override it in the context of the iframe. @domenic  do you have any comments how we can clarify this?